src/codeOfProgram/main/Main.java

```diff
@@ -0,0 +1,19 @@
+package codeOfProgram.main;
+
+import codeOfProgram.actions.AskingForAction;
+
+import java.util.List;
+
+public class Main {
+
+    public static void main(String[] args) throws Exception {
+        new Configurations();
+        List deckOfCards;
+
+        AskingForAction askingForAction = new AskingForAction();
+        deckOfCards = askingForAction.askingForStartGame();
```

Comment:
Do you really need this returned `deckOfCards` here?

Reply:
fixed